pkg/spire/spire_mock.go

```diff
@@ -24,9 +24,25 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	spireconfig "github.com/tektoncd/pipeline/pkg/spire/config"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+	"knative.dev/pkg/injection"
 )
 
+func init() {
+	injection.Fake.RegisterClient(withFakeControllerClient)
+	injection.Fake.RegisterClient(withFakeEntrypointerClient)
```

Comment:
This works with the mocked spire tests. But why don't you need the `GetEntrypointerAPIClient` here?